doc/guides/metricsAndAuditLogs.md

```diff
@@ -0,0 +1,69 @@
+# Displaying metrics and exploring audit logs
```

Comment:
Minor thing but headings should always be captialized with the exception of words like `the, and, of, for, in, at` etc. (unless they are at the beginning of the heading of course).

It should be `Displaying Metrics and Exploring Audit Logs`

Reply:
+1 done